_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,215 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the first HTML `title` element that 
+- is a [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) of a [document element](https://www.w3.org/TR/dom/#document-element) that is an HTML `html` element, and
+- contains [child text content](https://www.w3.org/TR/html/infrastructure.html#child-text-content) that does not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
I don't think my comment here has been addressed. This doesn't meet the plain language requirement IMO. "Child text content" and "unicode separator characters" are highly technical terms. Yes, we need these terms for an objective applicability, but we can still use plain language words and define those in a way that makes them objective.

Reply:
To be honest, I fail to see how hiding necessary complexity behind simpler words that are defined as something more complex is an improvement. In fact, it may not even end up being plain language: https://www.plainlanguage.gov/guidelines/words/minimize-definitions/. For starters, an improvement might be to go through this list and replace words where it makes sense: https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/